scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -0,0 +1,120 @@
+#!/usr//bin/env python
+
+"""
```

Comment:
We do not usually do not include in the scripts dates/people names, we can keep track of these things through github

Reply:
yes, I took it out